helpers/browser/driver.js

```diff
@@ -232,6 +238,59 @@ class ChromeProtocol {
       });
     });
   }
+
+  beginNetworkCollect() {
+    return this.getInstance().then(chrome =>
+      new Promise((resolve, reject) => {
+        this._networkRecords = [];
+        this._networkRecorder = new NetworkRecorder(this._networkRecords);
+
+        chrome.on('Network.requestWillBeSent',
```

Comment:
let's make a .listen([...]) and an unlisten so we can DRY this up, also unlisten() if passed 0 args should unregister everything